Dragonflight/DemonHunterHavoc.lua

```diff
@@ -927,6 +927,7 @@ spec:RegisterAbilities( {
             removeBuff( "restless_hunter" )
             applyBuff( "blade_dance" )
             setCooldown( "death_sweep", 15 * haste )
+            if level >= 22 then setCooldown( "blade_dance", 10 * haste ) end
```

Comment:
This isn't the right way to handle the cooldown change.  I'll take care of that part and manually integrate the other buff changes.

Reply:
Appreciate the feedback, looking forward to learning the proper way of doing these changes.